File: rb/test/exists_test.rb
# SpacexRest SDK exists test

require "minitest/autorun"
require_relative "../SpacexRest_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = SpacexRestSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- SpacexRest SDK exists test

local sdk = require("spacex-rest_sdk")

describe("SpacexRestSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
